.circleci/osx_install_dependencies.sh

```diff
@@ -43,13 +43,13 @@ then
   ./scripts/install_obsolete_jsoncpp_1_7_4.sh
 
   # z3
-  wget https://github.com/Z3Prover/z3/releases/download/z3-4.8.9/z3-4.8.9-x64-osx-10.14.6.zip
-  unzip z3-4.8.9-x64-osx-10.14.6.zip
-  rm -f z3-4.8.9-x64-osx-10.14.6.zip
-  cp z3-4.8.9-x64-osx-10.14.6/bin/libz3.a /usr/local/lib
-  cp z3-4.8.9-x64-osx-10.14.6/bin/z3 /usr/local/bin
-  cp z3-4.8.9-x64-osx-10.14.6/include/* /usr/local/include
-  rm -rf z3-4.8.9-x64-osx-10.14.6
+  wget https://github.com/Z3Prover/z3/releases/download/z3-4.8.10/z3-4.8.10-x64-osx-10.15.7.zip
+  unzip z3-4.8.10-x64-osx-10.15.7.zip
+  rm -f z3-4.8.10-x64-osx-10.15.7.zip
+  cp z3-4.8.10-x64-osx-10.15.7/bin/libz3.a /usr/local/lib
+  cp z3-4.8.10-x64-osx-10.15.7/bin/z3 /usr/local/bin
+  cp z3-4.8.10-x64-osx-10.15.7/include/* /usr/local/include
+  rm -rf z3-4.8.10-x64-osx-10.15.7
```

Comment:
This could really use a a variable :)

Reply:
true, I'll do that when I get the tests to pass